tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -41,7 +41,7 @@ class PipelineContext:
     PRODUCTION = bool(os.environ.get("PRODUCTION", False))  # Set this to True to enable production mode (e.g. to send PR comments)
 
     DEFAULT_EXCLUDED_FILES = (
-        [".git"]
+        [".git", "tools/ci_connector_ops/pipeline_reports/*", "tools/ci_connector_ops/ci_connector_ops/pipelines/*"]
```

Comment:
The pipeline code was not excluded from the containers mounting portions of the airbyte repo. It lead to cache bursting for each pipeline change.